src/services/utils/utils-service.ts

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
General suggestion. The term `service` is used in Angular. As for this SDK, there's no `component` `module`, so there's no need for the concept of `service`. We can just name them without the `-service` suffix.

For this `utils` and `request`, do we expect other files in the future to be in the same folder? If not, we'd better not create a folder for it, we can just put the file under `src` instead. Same with the `auth`, `registration` folder